src/assets/icons/audio-content.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <g id="icon">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M3.25 7L3.25 17H4.75V7H3.25Z" fill="currentColor"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M19.25 7V17H20.75V7H19.25Z" fill="currentColor"/>
```

Comment:
**small suggestion:** The `fill="currentColor"` could be indicated on the `g` tag and that way it is not necessary to repeat it on every `path`.